vmdb/app/controllers/ems_infra_controller.rb

```diff
@@ -33,11 +33,11 @@ def scaling
       return
     end
 
-    @count_parameters = @stack.parameters.select { |x| x.name.include?('::count') }
+    @count_parameters = @stack.parameters.select { |x| x.name.include?('Count') || x.name.include?('::count') }
 
     return unless params[:scale]
 
-    scale_parameters = params.select { |k, _v| k.include?('::count') }
+    scale_parameters = params.select { |k, _v| k.include?('Count') || k.include?('::count') }
```

Comment:
@rwsu it feels like this logic should be a method somewhere.  At a glance, I have no idea which one is OSP6 vs OSP7.  Do you have any ideas how we can move this logic down into a method with a good name?